data_handler/handlers/liquidable_debt/debt_handlers.py

```diff
@@ -49,45 +76,184 @@ def prepare_data(self, protocol_name: str, path: str = LOCAL_STORAGE_PATH) -> di
             url=self.connection_url
         )
         parsed_data = self._parse_file(uploaded_file_path)
+        sorted_parsed_data = self._sort_by_token_pair_correspondence(parsed_data)
+
+        return self._calculate_liquidable_debt(sorted_parsed_data)
 
-        return self._calculate_liquidable_debt(parsed_data)
+    @staticmethod
+    def _sort_by_token_pair_correspondence(
+            parsed_data: dict = None,
+    ) -> dict:
+        """
+        Sorts a given data by token pair correspondence.
+        Args: parsed_data( dict[
```

Comment:
Please, rework this docstring `Args` part with more readable description(example) of arg data